crates/core_arch/src/arm/neon.rs

```diff
@@ -1136,12 +1388,526 @@ pub unsafe fn vtbx4_p8(a: poly8x8_t, b: poly8x8x4_t, c: uint8x8_t) -> poly8x8_t
     ))
 }
 
+macro_rules! arm_vget_lane {
+    ($name:ident, $from:ty, $to:ty, $lanes:literal) => {
+        #[inline]
+        #[target_feature(enable = "neon")]
+        #[cfg_attr(target_arch = "arm", target_feature(enable = "v7"))]
+        #[cfg_attr(test, assert_instr(umov))]
+        pub unsafe fn $name(v: $from, lane: i32) -> $to {
```

Comment:
Same as above, can this be an `u32` ?

Reply:
This comment is resolved, but not change was made.

Reply:
Yes I closed all the 'same as above' lines and put a comment on the original one asking to see if u32 is a valid option. As mentioned in the initial comment it is not.